pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -4375,3 +4375,20 @@ func newBaseVmi(opts ...libvmi.Option) *v1.VirtualMachineInstance {
 	opts = append(opts, libvmi.WithResourceMemory("512Mi"))
 	return libvmi.New(opts...)
 }
+
+func newAdmissionReviewForVMICreation(vmi *v1.VirtualMachineInstance) (*admissionv1.AdmissionReview, error) {
```

Comment:
Please consider introducing this function with its first usage.
It would make its necessity clearer IMO.

Reply:
Done.